### DISCUSSION TITLE
How can i change the position of the ground and still have the collision effect?

### DISCUSSION THREAD

**Discussion by Tzikas** [enable3d/enable3d#268] [Help / Questions]
I create a ground object that holds my model fine, but once I changed the position of the ground object and the position of the model to the same new coordinates the ground/collision no longer works.  

```
//This works and the car doesn't fall through the ground  
const road = await this.third.physics.add.ground({ width: 20, height: 20 })
car.position.set(0,1,0)
this.third.scene.add(car)

//But once I move the road and the car, the ground no longer works and the car falls through it. 
const road = await this.third.physics.add.ground({ width: 20, height: 20 })
road.position.x = 20
car.position.set(20,1,0)
car.body.needUpdate = true
road.body.needUpdate = true
this.third.scene.add(car)
```

How can i change the position of the road and car and still have the collision work?

**Comment by yandeu** [comment]
I believe ground is static. So it can't be moved.

Please try a custom box with a kinematic body. Or set the collisionFlags of ground to be kinematic (2).

See https://enable3d.io/docs.html#physics-body
